src/main/java/com/timelogr/enterprise/dto/TimeLog.java

```diff
@@ -0,0 +1,20 @@
+package com.timelogr.enterprise.dto;
+
+import lombok.Data;
+
+public @Data
+class TimeLog {
+    private long id;
+    private long date; // IN UNIX TIMESTAMP
+    private long duration; // IN SECONDS
+
+    public TimeLog() {
+        super();
+    }
+
+    public TimeLog(long id, long date, long duration) {
+        this.id = id;
+        this.date = date;
+        this.duration = duration;
+    }
+}
```

Comment:
This comment is for all "new" classes. I simply renamed them to follow Java's class naming conventions found here:  [https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html) and I also included Project Lombok in these classes reduce boilerplate code, read more on Project Lombok here: [https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html](https://www.oracle.com/java/technologies/javase/codeconventions-namingconventions.html)

Reply:
Naming conventions and Lombok are both great ideas.